src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -363,6 +363,8 @@ public void OnFinishEditing()
         parent.AddChild(ReturnToStage);
         ReturnToStage.OnReturnFromEditor();
 
+        editedSpecies.UpdateInitialCompounds();
```

Comment:
I think this should be put after line 325, as it makes more sense to be there, than after performing the scene switch.

Reply:
Yep, moved.